app/elements/submission/sequential/sequential-challenge.html

```diff
@@ -43,8 +43,8 @@
 			margin: 0;
 			padding-top: 10px;
 		}
-		#leftFill{
-			width: 50px;
+		#leftProgressFlex{
+			margin-right: 50px;
```

Comment:
```css
#taskTimer{
 	margin: 0;
 	padding-top: 10px;
 	width: 50px;
 }
```

what i did is the counterpart of this. And why won`t this work on different screens? Do you know any screen that is smaller than 150px?